.circleci/calculate-parameters.py

```diff
@@ -34,11 +34,11 @@
 # How long should it take before an image is rebuilt.
 REBUILD_IMAGES_OLDER_THAN_DAYS = 7
 
-# Targets only built (and self-tested!) on Linux.
-LINUX_ONLY_TARGETS = ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu"]
+AARCH64_LINUX_BUILD_HOST = "aarch64-unknown-linux-gnu"
+X86_64_LINUX_BUILD_HOST = "x86_64-unknown-linux-gnu"
```

Comment:
Maybe make the `*_HOST`s lists as well? Then you don't need to wrap them in square brackets whenever you add them to another list.

PS. I know it is odd because it will only ever be a single element in the list, but it also avoids the bug (which you hit iirc) of accidentally adding two strings together.

```suggestion
AARCH64_LINUX_BUILD_HOST = ["aarch64-unknown-linux-gnu"]
X86_64_LINUX_BUILD_HOST = ["x86_64-unknown-linux-gnu"]
```

Reply:
I waffled on this for quite some time while writing it but I think you're right, and past-me made the wrong choice.

Notably, the `config.toml` treats `host` as an array, so we should too.